ThunderTable/ScrollOffsetManager.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ScrollOffsetManager.swift
+//  ThunderTable
+//
+//  Created by Simon Mitchell on 02/09/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import CoreGraphics
+import Foundation
+
+/// Scroll offset manager manages caching scroll offsets for UI in any scenario where
+/// scroll views may be re-used and so we need to store their current offset in memory
+/// to avoid re-use issues.
+class ScrollOffsetManager {
+    
+    private var offsetMap: [AnyHashable : CGPoint] = [:]
```

Comment:
Could doc it up for a bit of cleanliness/consistency, but this is very much personal preference.Your lass docs are sufficient explanation wise